normalize.css

```
@@ -342,8 +342,9 @@ template {
 
 /**
  * Add the correct display in IE 10.
+ * Except `hidden="until-found"`　because `content-visibility: hidden` is applied.
  */
 
-[hidden] {
+[hidden]:not([hidden="until-found"]) {
   display: none;
 }
```

`[hidden]:not([hidden="until-found"])` has [higher specificity](https://polypane.app/css-specificity-calculator/#selector=%5Bhidden%5D%3Anot(%5Bhidden%3D%22until-found%22%5D)) than the previous selector due to the `:not`. I wonder if this should instead be written as two separate declarations, so there is no increase in specificity? This will be less likely to cause issues for existing projects relying on `[hidden]` having a very low specificity.

```suggestion
[hidden] {
  display: none;
}

[hidden="until-found"] {
  display: revert;
}
```